Removing the images that are no longer being managed in azure-pipelin… (#1445)
microsoft/azure-pipelines-image-generation: images/macos/README.md

> The README.md files for the Azure pipelines Mac-based images are now in the [virtual-environments](https://github.com/actions/virtual-environments) repository. For future contributions or for issues regarding any of the images, please use the virtual-environments repository instead. You can still view the commit history of those images from when they used to live in the azure-pipelines-image-generation repository.

Reviewer: ```suggestion
The README.md files for the Azure Pipelines Mac-based images are now in the [virtual-environments](https://github.com/actions/virtual-environments) repository. Please submit issues in the virtual-environments repository instead.
```
